Neos.Neos/Resources/Private/Templates/Backend/Module/Index.html

```diff
@@ -73,13 +73,13 @@
 		<script src="{f:uri.resource(path: '2/js/bootstrap.min.js', package: 'Neos.Twitter.Bootstrap')}"></script>
 		<script src="{f:uri.resource(path: 'Library/fixedsticky/fixedsticky.js', package: 'Neos.Neos')}"></script>
 
-		<script type="text/javascript">
+		<script type="text/javascript"><![CDATA[
```

Comment:
Maybe not important here, but for good measure CDATA blocks should be commented in javascript
```suggestion
		<script type="text/javascript">/*<![CDATA[*/
```

Reply:
Not needed, the CDATA does not even arrive in the browser.